docs/benchmarks/_index.md

```diff
@@ -55,36 +57,58 @@ How to setup the environment for the Kafka Performance Test.
 
     ```bash
     helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com/
-    helm install --name zookeeper-operator --namespace zookeeper banzaicloud-stable/zookeeper-operator
+    helm install zookeeper-operator --namespace=zookeeper --create-namespace pravega/zookeeper-operator
     kubectl create -f - <<EOF
     apiVersion: zookeeper.pravega.io/v1beta1
     kind: ZookeeperCluster
     metadata:
-      name: example-zookeepercluster
+      name: zookeeper
       namespace: zookeeper
     spec:
       replicas: 1
     EOF
     ```
 
-1. Install the latest version of {{< kafka-operator >}}, the Operator for managing Apache Kafka on Kubernetes.
+1. Install the {{< kafka-operator >}} CustomResourceDefinition resources (adjust the version number to the Koperator release you want to install) and the corresponding version of {{< kafka-operator >}}, the Operator for managing Apache Kafka on Kubernetes.
+    ```bash
+    kubectl create --validate=false -f https://github.com/banzaicloud/koperator/releases/download/v0.22.0/kafka-operator.crds.yaml
```

Comment:
**Question**: Can we template the version here to latest?

Reply:
I don't know how we can point the CRDs to the generic latest version because the CRDs are published as assets of the tags, see https://github.com/banzaicloud/koperator/releases/tag/v0.22.0

Reply:
This was resolved by @fekete-robert 's commit https://github.com/banzaicloud/koperator-docs/pull/87/commits/efd62d17949e169cc7ecad0c2196d619d707afcf